interface/VideoEncoderDefs.h

```diff
@@ -151,6 +152,11 @@ typedef struct VideoFrameRate {
     uint32_t frameRateNum;
     uint32_t frameRateDenom;
 }VideoFrameRate;
+typedef struct VideoTemproalLayers {
+    //bitrate of the highest layer is VideoParamsCommon.rcParams.bitRate
+    uint8_t numLayersMinus1;
+    uint32_t bitRate[TEMPORAL_LAYER_LENGTH_MAX];
```

Comment:
Please change this to NUM_TEMPLAYER_MAX.
add add following comment
//layer 0 bitrate is bitRate[0]
//...
//layer highest birate is VideoParamsCommon.rcParams.bitRate